amz_review_scraper/get_soup.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+import requests
+import bs4
+import lxml
+
+from config import proxies, HEADER
+
+
+def boil_soup(url, asin):
+    for i in range(2):
+        raw_html = requests.get(url, headers=HEADER, proxies=proxies)
+        if raw_html.status_code == 200:
+            soup = bs4.BeautifulSoup(raw_html.text, "lxml")
+            return soup
+        else:
+            time.sleep(1)
+            print("Trying Again in 1 second")
```

Comment:
you want to reverse these, so it prints out it is sleeping and sleeping, instead of pausing mysterious, then telling you it was paused. 1 second isn't a big deal, but important in the future for longer sleep times.

Reply:
ooooo, i was def thinking of that wrong, i was thinking it triggered the 1 second then before it "completed" the second printed my line, which makes absolutely no sense when i think about it now.